picasso/src/main/java/com/squareup/picasso/Cache.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013 Square, Inc.
+ * Copyright (C) 2011 The Android Open Source Project
```

Comment:
this is out of date now, anyway, after Jesse's change to use Android's LruCache.
Should we do `Copyright (C) 2018 Square, Inc.`?